.circleci/config.yml

```diff
@@ -8,27 +8,26 @@ jobs:
           name: Install Dependencies
           command: |
             apt-get update
-            apt-get install cabal-install
+            apt-get install -y cabal-install
 
       - checkout
 
-      - run:
-          name: workaround for certificate failure
-          command: |
-            mv /etc/apt/sources.list.d/nodesource.list /etc/apt/sources.list.d/nodesource.list.disabled
-            apt-get update
-            apt-get -y upgrade
-            apt-get  install -y ca-certificates libgnutls30
-            mv /etc/apt/sources.list.d/nodesource.list.disabled /etc/apt/sources.list.d/nodesource.list
-
       - run:
           name: Install pip
           command: apt-get install -y python3-pip
 
       - run:
           name: Install dependencies
-          command: pip install --user -r requirements.txt
+          command: pip3 install --user -r requirements.txt
```

Comment:
```
❯ piptree ~/Work/haskell/haskell-spake2/requirements.txt 
attrs 21.4.0
spake2 0.8
└── hkdf
```

Let's add an hkdf pin to `requirements.txt` as protection against a future breaking version of hkdf.